gril/grilunsol.c

```diff
@@ -39,6 +39,32 @@
 #include "common.h"
 #include "grilunsol.h"
 
+int g_ril_unsol_parse_connected(GRil *gril, const struct ril_msg *message)
+{
+	struct parcel rilp;
+	int size;
+	int version = RIL_VERSION_UNSPECIFIED;
+
+	DBG("");
```

Comment:
I'm keeping this `DBG()` call there to sync with other unsol parsers. Removal of them are deferred.